trackbasket_be/app.py

```diff
@@ -3,6 +3,8 @@
 from flask_sqlalchemy import SQLAlchemy
 from models.volunteer import db, Volunteer
 from resources.volunteer import Volunteers
+from models.at_risk_user import db, AtRiskUser
+from resources.at_risk_user import AtRiskUsers
```

Comment:
thank you for following our conventions (plural for resources and singular for model)